client.go

```diff
@@ -277,6 +275,40 @@ func (c *Client) DoUpdate(ctx context.Context, creds Credentials) ([]byte, []byt
 	return result.Config, dhPrivkeyPEM, newCreds, nil
 }
 
+func (c *Client) CommandResponse(ctx context.Context, creds Credentials, responseToken string, response any) error {
+	value, err := json.Marshal(message.CommandResponseRequest{
+		ResponseToken: responseToken,
+	})
+	if err != nil {
+		return fmt.Errorf("failed to marshal DNClient message: %s", err)
+	}
+
+	jresp, err := json.Marshal(response)
+	if err != nil {
+		return fmt.Errorf("failed to marshal DNClient command response: %s", err)
+	}
+
+	postBody, err := SignRequestV1(message.CommandResponse, value, creds.HostID, creds.Counter, creds.PrivateKey)
+	if err != nil {
+		return nil
+	}
+	postBody = append(postBody, []byte("\n")...)
```

Comment:
CommandResponseRequest, I think, should wrap the one-shot response as part of the signed JSON response object. This code looks like it's appending the response object after the main response object?